spec/requests/api/v1/keywords_controller_spec.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe(Api::V1::KeywordsController) do
+  describe 'GET #index' do
+    context 'given one page of keywords' do
+      it 'returns all keywords' do
+        page_size = Pagy::DEFAULT[:items]
+        Fabricate.times(page_size, :keyword)
+
+        get(api_v1_keywords_path)
+
+        expect(json_response[:data].count).to(eq(page_size))
+      end
+    end
+
+    context 'given more than one page of keywords' do
+      it 'returns the first page of keywords' do
+        page_size = Pagy::DEFAULT[:items]
+        Fabricate.times(page_size + 1, :keyword)
+
+        get(api_v1_keywords_path)
+
+        expect(json_response[:data].count).to(eq(page_size))
+      end
+    end
+
+    context 'given a VALID page params' do
```

Comment:
English issue:

```suggestion
    context 'given VALID page params' do
```

Reply:
Fixed in 2b5e7442961bd7d26566fe5861be0fd2028457bf